code/datums/ruins/space_ruins.dm

```diff
@@ -326,3 +326,11 @@
 	description = "A cargo shuttle in a wrecked condition. There are many unknown horrors in space and looks like its last crew has faced one of them."
 	allow_duplicates = FALSE
 	cost = 1 // With the loot it contains it shouldn't be found frequently
+
+/datum/map_template/ruin/space/abandoned_engi_sat
+	id = "abandoned_engi_sat"
+	suffix = "abandoned_engi_sat.dmm"
```

Comment:
Yeah I've no idea why this is failing CI.

I checked this in the code and paths and its cased right.

Reply:
Whoops, fixed